documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -124,6 +124,14 @@ include::modules/mtv-ui.adoc[leveloffset=+2]
 include::modules/mtv-overview-page.adoc[leveloffset=+2]
 include::modules/mtv-settings.adoc[leveloffset=+2]
 
+[id="ways-to-create-migration-plans-ui"]
+=== Ways to create migration plans and components using the {ocp-short} web console
+
+There are two ways to create migration plans and components using the {ocp-short} web console:
+
+* The _step-by-step_ method in which you create plans components - such as providers and network mappings - and then, combine them into a migration plan. Although do not have to create the components in a definite order, the steps are listed below in the recommended order of work.
+* The _quick plan_ method in which you combine previously-defined components into a migration plan. This method is described in its own section, which follows the procedures for all of the steps in creating a migration plan.
```

Comment:
You don't need any "previously-defined components" - only the providers: source and target.
Mappings will be created "ad hoc" - similar to "create new mapping" option in the previous wizard.

Reply:
@rszwajko When i used the UI yesterday, I selected the VMs to migrate and MTV suggested the target provider (and the plan name and mappings). Is that OK?

Reply:
yes. thank you!